src/tests/run.sh

```diff
@@ -332,6 +336,9 @@ if [ $buildArch = "wasm" ]; then
     runtestPyArguments+=("-os" "Browser")
 fi
 
+if [ $buildOS = "Android" ]; then
+    runtestPyArguments+=("-os" "Android")
```

Comment:
Why is this needed now? Did this not work before when the script is supplied with `-os Android` before or was that never enabled?

Reply:
So for runtime tests, from terminal, you invoke `run.sh`, then `run.sh` invoke `runtest.py` to do more work. This is to pass OS configuration that you specified to `run.sh` down to `runtest.py`.